x-pack/plugins/canvas/canvas_plugin_src/functions/common/staticColumn.js

```diff
@@ -29,7 +27,8 @@ export const staticColumn = () => ({
   },
   fn: (context, args) => {
     const rows = context.rows.map(row => ({ ...row, [args.name]: args.value }));
-    const type = getType(rows[0][args.name]);
+    // typeof null is actually 'object', not 'null'.
+    const type = rows[0][args.name] === null ? 'null' : typeof rows[0][args.name];
```

Comment:
`getType` is using `== null`, which also matches undefined. Since `undefined` is not valid, and `null` is, we should probably keep that functionality.

aside: I don't understand the comment you added here, you're not using typeof...